workflows/quickstart/test/quickstart.test.js

```diff
@@ -0,0 +1,37 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const quickstart = require('../index');
+const {assert} = require('chai');
+
+const PROJECT_ID = process.env.GOOGLE_CLOUD_PROJECT;
+const LOCATION_ID = 'us-central1';
+const WORKFLOW = 'myFirstWorkflow';
+
+describe('Cloud Workflows Quickstart Tests', () => {
+  it('should execute the quickstart', async () => {
```

Comment:
This test and the HTTP requests inside the workflow don't appear to have any retry, meaning there is a flakiness risk related to the availability of the Cloud Function and Wikipedia API.

If there's something we can do to improve test reliability, it would be good.

Reply:
Yes, all system calls and network calls can fail.

After inspecting similar examples, I've added a `try` / `catch`.

For the Wikipedia workflow: The workflow handles retries of the workflow execution, not the client library.